observations/obs_converters/radar/README.rst

```diff
@@ -45,19 +45,119 @@ script:
 Real radar observations
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Once you have ensured that your data are quality controlled, use the 
-`Observation Processing And Wind Synthesis (OPAWS) <http://code.google.com/p/opaws/>`__
-utility convert your data to `obs_seq` format. The OPAWS utility reads specific
-types of files as input, such as DORADE sweep files and NSF NCAR EOL Foray data.
+The National Weather Service provides `NEXRAD (WSR-88D Type II) <https://www.roc.noaa.gov/level-two-data-types.php>`__
+radar data for download. We recommend the use of the 
+`Observation Processing And Wind Synthesis Python Tool (pyOPAWS) <https://github.com/louiswicker/pyOPAWS/>`__
+to directly convert from NEXRAD to DART `obs_seq` format.  This tool was created by Dr.
+Louis Wicker from NOAA, and also includes diagnostic scripting to visualize the data.  
+Follow the `installation <https://github.com/louiswicker/pyOPAWS?tab=readme-ov-file#installation>`__ 
+and `usage instructions <https://github.com/louiswicker/pyOPAWS?tab=readme-ov-file#usage-examples>`__ 
+to compile and run the pyOPAWS software.  We also provide quickstart instructions to compile and run pyOPAWS
+in the next section.  
```

Comment:
```suggestion
to compile and run the pyOPAWS software.  
```

Reply:
Per our conversation on Friday, this is a valid suggestion, but given current status at NOAA prefer to keep as is for now.